hackathon_site/event/jinja2/event/login.html

```diff
@@ -38,38 +37,41 @@
         >
         <label for="{{ form.password.id_for_label }}">Password</label>
         {% if form.password.errors %}
-            <span class="formFieldError">
+            <span class="formFieldError col s12">
             {% for error in form.password.errors %}
                 {{ error }}
                 <br />
             {% endfor %}
             </span>
         {% endif %}
     </div>
+    {% if form.non_field_errors %}
+        {% for error in form.non_field_errors() %}
+            <div class="formFieldError col s12">
```

Comment:
col inside a col. Also, make this `formError` class which just doesn't have tiny text